File: src/main/java/io/github/dqualizer/dqlang/types_new/adapter/k6/Stage.java
package io.github.dqualizer.dqlang.types_new.adapter.k6;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Stage {

    private String duration;
    private int target;
}

File: src/main/kotlin/dqualizer/dqlang/archive/k6configurationrunner/dqlang/options/Stage.kt
package dqualizer.dqlang.archive.k6configurationrunner.dqlang.options

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class Stage {
    private val duration: String? = null
    private val target = 0
}
